code/modules/research/mechanic/flatpack.dm

```diff
@@ -264,3 +264,9 @@
 
 #undef Fl_ACTION
 */
+
+
+/obj/structure/closet/crate/flatpack/suit_modifier/New()
+	..()
+	machine = new /obj/machinery/suit_modifier(src)
+	new /obj/item/rig_module/health_readout(src)
```

Comment:
Is there a reason this comes with a freebie? Was it for testing purposes?

Reply:
Might be seeing as the healthbar was the big standard thing for RIGs in Dead Space, but that's just what I thought of.

Reply:
Bit of both